modules/core/include/opencv2/core/hal/intrin_forward.hpp

```diff
@@ -188,4 +188,4 @@ CV_CPU_OPTIMIZATION_HAL_NAMESPACE_END
 
 //! @endcond
 
-} // cv::
\ No newline at end of file
+} // cv::
```

Comment:
Unfortunately I can't unroll this changes. Visual Studio insert shift to next line automatically.

Reply:
BTW, `git` can unroll any file. We don't need editor for that.

```
git checkout upstream/master -- modules/core/include/opencv2/core/hal/intrin_forward.hpp
```

Reply:
@alalek Ok. Done.